### ISSUE TITLE
Link CCL to CAMB

### ISSUE THREAD

**Issue by elisachisari** [LSSTDESC/CCL#226]
Consider an using CAMB for getting the power spectrum

**Comment by philbull** [comment]
CosmoSIS has a CAMB driver if I remember correctly. Should we use that rather than implementing our own?

**Comment by tmcclintock** [comment]
In [this thread](https://github.com/lesgourg/class_public/issues/189) on the CLASS issues page I posted the same plot I did in the slack channel showing the difference between the linear matter power spectra calculated with CAMB vs CLASS for a very basic wCDM cosmology. The difference isn't negligible for observables, but it's just barely on the edge of what we will care about in LSST Y1.

Thomas Tran gave be advice about what I could change in CLASS to get it to agree with CAMB. This begs the question - what kind of consistency do we want to promise the user? We can spend time hacking one Boltzmann code to agree with the other, or should we just say "You will see differences of 1% for some models"?
